cudapoa/include/claragenomics/cudapoa/utils.hpp

```diff
@@ -0,0 +1,57 @@
+/*
+* Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+*
+* NVIDIA CORPORATION and its licensors retain all intellectual property
+* and proprietary rights in and to this software, related documentation
+* and any modifications thereto.  Any use, reproduction, disclosure or
+* distribution of this software and related documentation without an express
+* license agreement from NVIDIA CORPORATION is strictly prohibited.
+*/
+
+#pragma once
+
+#include <vector>
+#include "batch.hpp"
```

Comment:
let's make this `#include <claragenomics/cudapoa/batch.hpp>`

Reply:
done.